app/_includes/prereqs/ollama.md

```diff
@@ -0,0 +1,47 @@
+To complete this tutorial, make sure you have Ollama installed and running locally.
+
+1. Visit the [Ollama download page](https://ollama.com/download) and download the installer for your operating system. Follow the installation instructions for your platform.
+
+1. Start Ollama:
+   ```sh
+   ollama start
+   ```
+
+2. After installation, open a new terminal window and run:
+
+   ```sh
+   ollama run llama2
+   ```
+
+   You can replace `llama2` with the model you want to run, such as `llama3`.
+
+3. To set up the AI Proxy plugin, you'll need the upstream URL of your local Llama instance. 
+
+   ```sh
+   export DECK_OLLAMA_UPSTREAM_URL='http://localhost:11434'
```

Comment:
```suggestion
   export DECK_OLLAMA_UPSTREAM_URL='http://localhost:11434/api/chat'
```
Mentioned it down the page, but the `/api/chat` will have to be part of the variable for it to work in decK